.github/workflows/build.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           distribution: 'temurin'
       - name: Build and test
         run: |
+          mvn --batch-mode org.apache.maven.plugins:maven-dependency-plugin:3.2.0:go-offline
```

Comment:
I will cherry-pick these back to main (and add a step timeout), and probably keep fighting with whatever network issue github actions is having:

```
[INFO] ------------------------------------------------------------------------
[INFO] Building J2CL Maven Archetypes 0.19-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-resources-plugin/maven-metadata.xml
Jan 06, 2022 8:32:55 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {s}->https://repo.maven.apache.org:443: Connection timed out (Read failed)
Jan 06, 2022 8:32:56 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://repo.maven.apache.org:443
Jan 06, 2022 8:48:40 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {s}->https://repo.maven.apache.org:443: Connection timed out (Read failed)
Jan 06, 2022 8:48:40 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://repo.maven.apache.org:443
Jan 06, 2022 9:04:24 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.SocketException) caught when processing request to {s}->https://repo.maven.apache.org:443: Connection timed out (Read failed)
Jan 06, 2022 9:04:24 PM org.apache.maven.wagon.providers.http.httpclient.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://repo.maven.apache.org:443
Warning:  Could not transfer metadata org.apache.maven.plugins:maven-resources-plugin/maven-metadata.xml from/to central (https://repo.maven.apache.org/maven2): Connection timed out (Read failed)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] J2CL Build Tools ................................... SUCCESS [  2.545 s]
[INFO] Build caching tools ................................ SUCCESS [  5.239 s]
[INFO] J2CL Cachable builds ............................... SUCCESS [ 13.839 s]
[INFO] J2CL Maven Plugin .................................. SUCCESS [04:53 min]
[INFO] J2CL Maven Archetypes .............................. FAILURE [  01:02 h]
[INFO] Simple J2CL Project Archetype ...................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:08 h
[INFO] Finished at: 2022-01-06T21:20:08+00:00
[INFO] Final Memory: 37M/127M
[INFO] ------------------------------------------------------------------------
```

Reply:
Looks like this can be dropped now, GHA isn't breaking on each build any more...

Worth following up later with a way to do this, but maven was being irritating as I tried various options.